rancid-tomatillos/src/Components/MovieView/MovieView.js

```diff
@@ -7,24 +7,26 @@ const MovieView = ({selectedMovie}) => {
     return(
         <div className="movieViewMain" style={{ backgroundImage: `url(${selectedMovie.backdrop_path})` }}>
             <div className="posterTitleContainer">
-                <h2 className="title">{selectedMovie.title}</h2>
-        <div className="statsContainer">
-            <p className="releaseDate">{selectedMovie.release_date}</p>
-            <p className="tagline">{selectedMovie.tagline}</p>
-            <p className="rating">{selectedMovie.average_rating} ⭐️</p>
-            <p className="budget">$ {selectedMovie.budget}</p>
-            <p className="revenue">$ {selectedMovie.revenue}</p>
-            <p className="runtime">{selectedMovie.runtime} mins.</p>
+                <div className="floater">
```

Comment:
Thank you for adding these! I hope these will help once we work on our routing in the next iteration. It will be good to see where everything ends up. 🎬